pyne/mcnp.py

```diff
@@ -1425,9 +1441,12 @@ def mat_from_inp_line(filename, mat_line, densities='None'):
                 nucvec[zzzaaam] += float(data_string.split()[i+1])
             else:
                 nucvec[zzzaaam] = float(data_string.split()[i+1])
+            
 
             if len(data_string.split()[i].split('.')) > 1:
                 table_ids[str(zzzaaam)] = data_string.split()[i].split('.')[1]
+            else:
+                table_ids[str(zzzaaam)] = lib_id
```

Comment:
These IDs are attributes of the whole material, not each nuclide.  For example, if someone manipulated the material to have addition nuclides (e.g. depletion/transmutation) we'd want the new nuclides to inherit this info.

They can be added as separate metadata, or as a stand alone entry in `table_ids`.  
E.g: `table_ids['default'] = lib_id` at some location that is not nuclide specfic, like when `table_ids` is first declared, above

Also, it should only be used if it's not empty.

Reply:
I thought that to assign a default library identifier to each isotope would be more useful in the future pyNE releases. But, as you said, since all isotopes in a material card would use the default libraries, if exists, we can specify the lib as a separate metadata. So, your point definitely makes sense. In this case, I will try to add `lib_id` in `table_ids` with `default` key name in order to be more specific.

Reply:
Solved!